.zenodo.json

```diff
@@ -0,0 +1,95 @@
+{
+  "title": "MNE-BIDS: An MNE project for organizing and formatting MEG and EEG data according to the BIDS specification.",
+  "description": "MNE-BIDS is a Python library to facilitate the analysis of MEG and EEG with MNE-Python for data conforming to the BIDS (Brain Imaging Data Structure) format. More information about BIDS and MNE can be found under <a href=\"https://bids.neuroimaging.io\">bids.neuroimaging.io</a> and <a href=\"https://mne.tools\">mne.tools</a> respectively.",
+  "creators": [
+    {
+      "affiliation": "Center for Adaptive Rationality, Max Planck Institute for Human Development, Berlin, Germany",
+      "name": "Appelhoff, Stefan",
+      "orcid": "0000-0001-8002-0877"
+    },
+    {
+      "affiliation": "",
+      "name": "Mainak Jas",
+      "orcid": "0000-0002-3199-9027"
+    },
+    {
+     "affiliation": "",
+      "name": "monkeyman192",
+      "orcid": "xxxx-xxxx-xxxx-xxxx"
+    },
+    {
+     "affiliation": "",
+      "name":"Teon Brooks",
+      "orcid": "xxxx-xxxx-xxxx-xxxx"
+    },
+    {
+     "affiliation": "",
+      "name": "wmvanvliet",
+      "orcid":
+    },
+    {
+     "affiliation": "",
+      "name": "romquentin",
+      "orcid": "xxxx-xxxx-xxxx-xxxx"
+    },
+    {
+     "affiliation": "",
+      "name": "Chris Holdgraf",
+      "orcid": "0000-0002-2391-0678"
+    },
+    {
+     "affiliation": "",
+      "name": "dnacombo",
+      "orcid": "xxxx-xxxx-xxxx-xxxx"
+    },
+    {
+     "affiliation": "",
+      "name": "ezemikulan",
```

Comment:
affilitation: University of Milan, Milan, Italy
name: Ezequiel Mikulan
orcid: 0000-0001-7259-6120